pkg/controller/elasticsearch/client/client.go

```diff
@@ -71,9 +71,11 @@ type Client interface {
 	DisableReplicaShardsAllocation(ctx context.Context) error
 	// EnableShardAllocation enables shards allocation on the cluster.
 	EnableShardAllocation(ctx context.Context) error
-	// SyncedFlush requests a synced flush on the cluster.
+	// SyncedFlush requests a synced flush on the cluster. Deprecated in 7.6, removed in 8.0.
 	// This is "best-effort", see https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-synced-flush.html.
 	SyncedFlush(ctx context.Context) error
+	// Flush requests a flush on the cluster.
```

Comment:
If you're bored you can try to repeat this comment several times as quickly as possible. It's very hard to pronounce!
(I'm getting ready for April 1st)